06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(csv_file):
+    data = {}
+    with open(csv_file, "r") as csv_file:
+        try:
+            csv_reader = csv.reader(csv_file)
+            for name, score in csv_reader:
+                score = int(score)
+                data[name] = max(data.get(name) or 0, score)
+        except ValueError as e:
+            print("Could not parse csv", end="")
+            print(e)
```

Comment:
As my comment from the previous round, you should return `None` in case of an error instead of letting the flow continue normally and reach `return data`.